addon/bootstrap.js

```diff
@@ -1,17 +1,65 @@
-/* globals Components, AddonManager */
+/* globals AddonManager, Components, Services */
 /* eslint-disable no-unused-vars */
 
 const OLD_ADDON_PREF_NAME = "extensions.jid1-NeEaf3sAHdKHPA@jetpack.deviceIdInfo";
 const OLD_ADDON_ID = "jid1-NeEaf3sAHdKHPA@jetpack";
+const ADDON_ID = "pageshot@mozilla.org";
+const TELEMETRY_PREF = "toolkit.telemetry.enabled";
+const USER_ENABLE_PREF = "extensions.pageshot.userdisabled";
+const SYSTEM_ENABLE_PREF = "extensions.pageshot.systemdisabled";
```

Comment:
Well, this is my fault, but these variables have a really bad name – ENABLE when in fact they are DISABLE